contracts/src/v0.8/vrf/VRFCoordinatorV2Plus.sol

```diff
@@ -4,13 +4,13 @@ pragma solidity ^0.8.4;
 import "../interfaces/LinkTokenInterface.sol";
 import "../interfaces/BlockhashStoreInterface.sol";
 import "../interfaces/AggregatorV3Interface.sol";
-import "../interfaces/TypeAndVersionInterface.sol";
+import "../interfaces/IVRFMigratableCoordinatorV2Plus.sol";
 import "./VRF.sol";
 import "./VRFConsumerBaseV2Plus.sol";
 import "../ChainSpecificUtil.sol";
 import "./SubscriptionAPI.sol";
 
-contract VRFCoordinatorV2Plus is VRF, TypeAndVersionInterface, SubscriptionAPI {
+contract VRFCoordinatorV2Plus is VRF, SubscriptionAPI {
```

Comment:
Doesn't seem to be necessary because `onMigration()` is not supported in this version of the coordinator. Let me know if you think otherwise though. 
I think we do need a narrower interface that the consumers use to call `requestRandomWords()`. `IVRFCoordinatorV2Plus` is too broad right now and some functions in that interface may not be supported in the future versions.